### ISSUE TITLE
Sistema de VIPs para o PerfectDreams.

### ISSUE THREAD

**Issue by BsouzaM** [PerfectDreams/StickyNotes#15]
## 🤔 Antes de enviar...
Para manter tudo organizado, siga estes passos antes de começar:
- [X] Eu já confirmei que esta sugestão será útil para outros usuários.
- [X] Eu já procurei por sugestões parecidas no repositório.
- [X] Eu já verifiquei que não existe nenhuma outra branch ou pull request aberto com a minha sugestão ou com uma sugestão parecida.
- [ ] Eu tentei procurar por [outros repositórios do PerfectDreams](https://github.com/PerfectDreams) um repositório relacionado com a minha sugestão, mas não encontrei/é uma sugestão sobre algo que não tem nada relacionado no servidor.

### 🎮 Seu nome no PerfectDreams
KneezMz

### ✨ Descrição da sugestão
A sugestão é dividida em três partes. Sendo elas: Como poderia funcionar a tag VIP, como vou me locomover de um ponto ao outro já que tudo é longe e o que irá me manter entretido no servidor, mas está dividida nos issues.

### 🌟 Como irá funcionar?
Para começar, o MrPowerGamerBR havia falado que estava em dúvida de como ia resolver o VIP, pois bem. Aqui está minha ideia:

- Achei que seria interessante uma opção de tempo para vip, por exemplo de 30 dias, 60 dias, 90 dias, 120 dias, 150 dias, permanente e também uma forma de doação de qualquer valor para ajudar o servidor sem (ou com) recompensas;
- Por default, o nome branco é o player sem vip.
- No VIP, o player pode optar por escolher o nome com qualquer cor;
- Para simplificar todas as tags de VIP, VIP+, VIP++, VIP++++++ e por aí vai, decidi que apenas um VIP basta. Na vida real não existe VIP+ ou VIP++. Existe VIP.

### 😎 Porque ela seria legal no PerfectDreams?
Porque simplifica todas essas tags grandes e desnecessárias, embora seria mais interessante colocá-las ao passar o mouse por cima do nome do player.

**Comment by Pokeer** [comment]
Então o VIP teria todas as vantagens que um VIP++ teria? 🤔

**Comment by BsouzaM** [comment]
@Pokeer 
> Então o VIP teria todas as vantagens que um VIP++ teria? thinking:

Sim, e para complementar, poderia ter mais recompensas caso você opte por mais meses. Imagine como se alguém tivesse apenas uma escolha em simplificar todas essas tags. Como você definiria a de VIP? Eu definiria assim.

**Comment by MrPowerGamerBR** [comment]
Na verdade o problema é os preços

**Comment by BsouzaM** [comment]
@MrPowerGamerBR 
> Na verdade o problema é os preços

Eu estava pensando nisso a pouco tempo e imaginei como a seguinte frase: "Quanto menor o preço, mais gente tem"

E nisso eu adaptei para os meses: 30 dias, o mais barato de todos (sendo sempre VIP, não +, ++, etc. para todos os meses), permanente o mais caro.
